app/javascript/controllers/viral/sortable_lists/two_lists_selection_controller.js

```diff
@@ -20,33 +17,51 @@ export default class extends Controller {
   connect() {
     this.availableList = document.getElementById(this.availableListValue)
```

Comment:
**suggestion:** Wouldn't these be better as stimulus targets?  That way you would not have to query the DOM to get them.

Reply:
As discussed, we'll leave this as is